src/MTGCore.ClientApp/ClientApp/src/App.vue

```diff
@@ -44,138 +45,156 @@
+      this.$refs.SearchFilterRef.open();
+    },
     addNewDeck(title) {
       var params = new URLSearchParams();
       params.append('title', title);
 
       axios({
-        method: "post",
+        method: 'post',
         url: `https://localhost:44305/api/Deck/New`,
-        data: params
+        data: params,
       })
-          .then(res => {
-            this.updateDeckCardList(res.data);
-            this.updateDeckList();
-            // Set selected dropdown value to new deck ID
-            this.selectedDeck = res.data;
-            // Set textbox val back to empty string
-            this.$refs.deckListRef.resetNewDeckName();
-          })
-          .catch(err => {
-            alert(`There was an error submitting your form. See details: ${err}`);
-          });
+        .then((res) => {
+          this.updateDeckCardList(res.data);
+          this.updateDeckList();
+          // Set selected dropdown value to new deck ID
+          this.selectedDeck = res.data;
+          // Set textbox val back to empty string
+          this.$refs.deckListRef.resetNewDeckName();
+        })
+        .catch((err) => {
+          alert(`There was an error submitting your form. See details: ${err}`);
+        });
     },
     addToDeck(id) {
       axios({
-        method: "post",
+        method: 'post',
         url: `https://localhost:44305/api/Deck/Add/${this.selectedDeck}/${id}`,
-        data: this.$data
+        data: this.$data,
       })
-          .then(() => {
-            this.updateDeckCardList(this.selectedDeck);
-          })
-          .catch(err => {
-            alert(`There was an error submitting your form. See details: ${err}`);
-          });
+        .then(() => {
+          this.updateDeckCardList(this.selectedDeck);
+        })
+        .catch((err) => {
+          alert(`There was an error submitting your form. See details: ${err}`);
+        });
     },
     SubmitForm() {
       this.searchInProgress = true;
       this.searchedCards = null;
       this.searchedCardsLoadingError = null;
+
+
+      var filter = {}
+      filter["SearchFilter"] = this.searchFilter;
+      filter["SearchFilter"].Name = this.Name;
+
       axios({
-        method: "post",
-        url: "https://localhost:44305/api/Search/",
-        data: this.$data
+        method: 'post',
+        url: 'https://localhost:44305/api/Search/',
+        data: filter,
       })
-          .then(res => {
-            this.searchedCards = res.data;
-          })
-          .catch(err => {
-            this.searchedCardsLoadingError = `Something went wrong: ${err}`;
-          })
-          .finally(() => {
-            this.searchInProgress = false;
-          });
+        .then((res) => {
+          this.searchedCards = res.data;
+        })
+        .catch((err) => {
+          this.searchedCardsLoadingError = `Something went wrong: ${err}`;
+        })
+        .finally(() => {
+          this.searchInProgress = false;
+        });
```

Comment:
This will cause some major conflicts when merging due to the last PR. It might be worth removing unessential changes to make your life easier when it comes to merging.